README.md

```diff
@@ -130,50 +130,54 @@ client.task('5231c687bbdf0f94fa000007').result('523c0ee0ca452f0000000009').get({
-
-Now you'll be able to run the following commands:
+If you'd like to contribute code, get started by cloning the repo and running `npm install`. Now you'll be able to run the following commands. Please use these build tools to avoid your contribution being delayed by a lint error or a failing test:
 
 ```sh
-$ grunt       # Run the lint and test tasks together
-$ grunt lint  # Run JSHint with the correct config
-$ grunt test  # Run unit tests
+# Lint your contribution
+grunt lint
 ```
 
-Code with lint errors or failing tests will not be accepted, please use the build tools outlined above.
+```sh
+# Test your contribution
+grunt test
+```
 
-For users with push-access, don't commit to the master branch. Code should be in `develop` until it's ready to be released.
+```sh
+# Lint and test with a single command
+grunt
+```
 
 ## Support and Migration
 
-Major versions are normally supported for 6 months after their last minor release. This means that patch-level changes will be added and bugs will be fixed. The table below outlines the end-of-support dates for major versions, and the last minor release for that version.
+We maintain a [migration guide](MIGRATION.md) to help you migrate between major versions.
 
-We also maintain a [migration guide](MIGRATION.md) to help you migrate.
+A previous major version of this client will be supported for a minimum of 6 months from its final minor release. During this time bugs will continue to be fixed and released in the form of patch versions. If you're opening an issue related to this project, please mention the specific version that the issue affects.
 
-| :grey_question: | Major Version | Last Release | Node.js Versions | Support End Date |
-| :-------------- | :------------ | :----------- | :--------------- | :--------------- |
-| :heart:         | 3             | N/A          | 12+              | N/A              |
-| :hourglass:     | 2             | 2.0.0        | 8+               | 2022-05-26       |
-| :skull:         | 1             | 1.2.1        | 0.10+            | 2020-01-05       |
+The following table lists the major versions available and, for each previous major version, its end-of-support date, and its final minor version released.
 
-If you're opening issues related to these, please mention the version that the issue relates to.
+| Major version | Last release | Node.js required | End of support                           |
+| :------------ | :----------- | :--------------- | :--------------------------------------- |
+| 4             | 4.0.0        | 14+              | :white_check_mark: Current major version |
+| 3             | 3.0.0        | 12+              | 2023-11-01                               |
+| 2             | 2.0.0        | 8+               | 2022-05-26                               |
+| 1             | 1.2.1        | 0.10+            | 2020-01-05                               |
```

Comment:
Removed the first column because the truth of ⌛ is affected by the passage of time

Reply:
Good point. One thing about the end date though, we tend to follow SN's policy of (trying to) support older versions for 6 months, as described in the Playbook:
https://github.com/springernature/frontend-playbook/blob/main/practices/open-source-support.md#support-for-old-versions

In this case, it would mean that v3 would have a EOL date of 2024-04-xx, or so.

Reply:
Thanks @josebolos.  As we've discussed elsewhere I'll strip this section's change out for now and we can revisit it in a further PR before publishing v4.